src/screens/SelectOrganization/SelectOrganization.tsx

```diff
@@ -0,0 +1,60 @@
+import { useQuery } from '@apollo/client';
+import AdminNavbar from 'components/AdminNavbar/AdminNavbar';
+import OrganizationCardStart from 'components/OrganizationCardStart/OrganizationCardStart';
+import { USER_ORGANIZATION_LIST } from 'GraphQl/Queries/Queries';
+import React from 'react';
+import styles from './SelectOrganization.module.css';
+
+function SelectOrganizationPage(): JSX.Element {
+  const a = window.location.href.split('=')[1];
```

Comment:
"a" is **not** a descriptive name. Please use a descriptive name for your variable declaration